packages/jupyter-ai/jupyter_ai/chat_handlers/base.py

```diff
@@ -48,6 +48,29 @@ class SlashCommandRoutingType(HandlerRoutingType):
     underscores."""
 
 
+class MarkdownHelpFormatter(argparse.ArgumentDefaultsHelpFormatter):
+    def _get_help_string(self, action):
+        # do not show "(default: False)" for flags as this is assumed
+        if action.const is True:
+            return action.help
+        return super()._get_help_string(action)
+
+    def _format_action_invocation(self, action):
+        if not action.option_strings:
+            return super._format_action_invocation(action)
```

Comment:
When calling `/learn -h`, I get an error:

```
Traceback (most recent call last):
  File "/Volumes/workplace/jupyter-ai/packages/jupyter-ai/jupyter_ai/chat_handlers/base.py", line 162, in on_message
    await self.process_message(message)
  File "/Volumes/workplace/jupyter-ai/packages/jupyter-ai/jupyter_ai/chat_handlers/learn.py", line 132, in process_message
    self.reply(self.parser.format_help(), message)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dlq/micromamba/envs/jai/lib/python3.11/argparse.py", line 2588, in format_help
    formatter.add_arguments(action_group._group_actions)
  File "/Users/dlq/micromamba/envs/jai/lib/python3.11/argparse.py", line 281, in add_arguments
    self.add_argument(action)
  File "/Users/dlq/micromamba/envs/jai/lib/python3.11/argparse.py", line 266, in add_argument
    invocations = [get_invocation(action)]
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/Volumes/workplace/jupyter-ai/packages/jupyter-ai/jupyter_ai/chat_handlers/base.py", line 60, in _format_action_invocation
    return super._format_action_invocation(action)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: type object 'super' has no attribute '_format_action_invocation'
```

```suggestion
            return super()._format_action_invocation(action)
```

Reply:
Oops, sorry about that - silly typo when refactoring code!